packages/lib/utils/httpPrefix.ts

```diff
@@ -0,0 +1,12 @@
+// Check that a given URL starts with "http://" or "https://". If not, prefix it with "https://".
+// @param url The URL to check.
+// @returns The given URL if it starts with "http://" or "https://", and if that is not the case, add "https://" as a prefix.
+function httpPrefix(url: string) {
```

Comment:
```suggestion
const addHttpPrefix = (url: string) => {
```

(Please see coding style - we should only create arrow functions. And the function should tell what it does)